docker-compose.storage.minio.yml

```diff
@@ -7,7 +7,7 @@ services:
     minio:
         # DO NOT upgrade this to any version "2021.05" or later.
         # We want to stay on Apache license for now.
-        image: minio/minio:RELEASE.2019-04-23T23-50-36Z
+        image: minio/minio:RELEASE.2021-04-22T15-44-28Z
```

Comment:
This should be on a separate commit or mentioned in the commit body of the single commit.